index.bs

```diff
@@ -10956,10 +10950,8 @@ is the concatenation of the [=interface=]’s
     is called with property key |P|, the following steps are taken:
 
     1.  Let |A| be the [=interface=] for the [=named properties object=] |O|.
-    1.  Let |object| be the sole object from |O|’s ECMAScript global environment that [=implements=] |A|.
-
-        Note: For example, if the [=interface=] is the {{Window}} interface,
-        then the sole object will be this global environment’s window object.
+    1.  Let |object| be |O|.\[[Realm]]'s [=Realm/global object=].
```

Comment:
You could also use O's relevant global object. But, your version is probably a bit better, as it stays closer to the IDL/ES level.